### DISCUSSION TITLE
Can I use grid like storybook sample?

### DISCUSSION THREAD

**Discussion by reuse-ay** [maxGraph/maxGraph#408] [Q&A] [answered]
[like this.](https://maxgraph.github.io/maxGraph/demo/?path=/story/connections-anchors--default)
Or does this come from storybook?

**Comment by tbouffard** [comment, marked as answer]
yes you can
the style of the container used to display the graph is set to use a background image.
See https://github.com/maxGraph/maxGraph/blob/v0.10.1/packages/html/stories/shared/configure.js#L23-L37
The grid image match the default grid size: https://github.com/maxGraph/maxGraph/blob/96055892a634b702d78f1c21fdaa12fc6177640b/packages/core/src/view/mixins/SnapMixin.ts#L69-L73

It is also possible to build a dynamic grid, which requires some code
- live story (push the -/+ button at least one to display the grid): https://maxgraph.github.io/maxGraph/demo/?path=/story/backgrounds-grid--default
- code: https://github.com/maxGraph/maxGraph/blob/v0.10.1/packages/html/stories/Grid.stories.js

**Reply by reuse-ay** [reply]
Thank you!
